From 77af028484c50bf1c817bc5a48d2568cc1d446fa Mon Sep 17 00:00:00 2001
From: wangxiuqi <wangxiuqi@126.com>
Date: Thu, 23 Jul 2020 22:56:28 +0800
Subject: [PATCH] change 2 files

---
 test.c  |  1 +
 test1.c | 14 ++++++++------
 2 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/test.c b/test.c
index ba8ebe0..abb942a 100644
--- a/test.c
+++ b/test.c
@@ -2,4 +2,5 @@
 void main(void)
 {
 	printf("user");
+	//patch1 0723
 }
diff --git a/test1.c b/test1.c
index 28ec695..e4f69ee 100644
--- a/test1.c
+++ b/test1.c
@@ -1,10 +1,12 @@
+void test1()
+{
+//master_test
 
-master_test
-
-branch1_test add some text 22:34
+//branch1_test add some text 22:34
  
-branch2_test "add new text" 30300630 09:32 sda fds
+//branch2_test "add new text" 30300630 09:32 sda fds
 
 
-"add by wwx12:37"
-"add by wwx12:43"
+//"add by wwx12:37"
+//"add by wwx12:43"
+}
\ No newline at end of file
-- 
2.27.0.windows.1

